cmd/nfd-master/main.go

```diff
@@ -117,7 +117,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 	flagset.StringVar(&args.Kubeconfig, "kubeconfig", "",
 		"Kubeconfig to use")
 	flagset.BoolVar(&args.EnableNodeFeatureApi, "enable-nodefeature-api", true,
-		"Enable the NodeFeature CRD API for receiving node features. This will automatically disable the gRPC communication.")
+		"Enable the NodeFeature CRD API for receiving node features. This will automatically disable the gRPC communication."+
+			"DEPRECATED: will be removed in a future release.")
```

Comment:
small suggestion to chage:

```suggestion
			"DEPRECATED: will be removed in a future release along with the deprecated gRPC API")
```

WDYT?